js/interface.js

```diff
@@ -276,7 +276,7 @@
           return;
         }
 
-        currentProvider = Fliplet.Widget.open(currentMenu.id);
+        currentProvider = Fliplet.Widget.open(currentMenu.instances[0].id);
```

Comment:
This seems significantly different from what you had before? Either this is wrong or the last implementation was wrong. If the last implementation was wrong, this shows a lack of testing before submitting for PR review.

Reply:
@tonytlwu, I tested all scripts before this fix and everything worked correctly, the error was that when I changed the destination to CurrentMenu, I did not notice that elsewhere this variable is used and affects another script, sorry I didn’t notice, it works now correct as it should

Reply:
@AndrRyaz If you're making changes and was ready to submit the PR with an error, then it shows that testing wasn't complete. If this was approved at that stage and deployed to staging (or production), it would only create more bugs and further reviews & delays. It shouldn't matter whether you noticed the code needed changing -- it's the testing that should be more thorough before submitting for a PR review cc @nishdandamudi

Reply:
@tonytlwu You are right, I missed this point, but in the future I will be more careful with the data that I change so that no scenario is broken